source/outlines/2016-03-10.markdown

```diff
@@ -0,0 +1,120 @@
+---
+title: 20160310
+layout: outline
+---
+
+## All
+
+### Word of the Day
+
+* bowdlerize - transitive verb - to remove or modify the parts (of a book,
+for example) considered offensive.
+
+
+## 1508
+
+### Portfolio Reviews (9:00 - WE FINISH)
+
+In this order:
```

Comment:
I'll add Jess in. I would keep it alphabetical because just to avoid the conspiracy theories.